application/src/main/java/org/thingsboard/server/service/sync/vc/repository/RepositorySettingsRedisCache.java

```diff
@@ -31,6 +31,6 @@
 public class RepositorySettingsRedisCache extends RedisTbTransactionalCache<TenantId, RepositorySettings> {
 
     public RepositorySettingsRedisCache(TBRedisCacheConfiguration configuration, CacheSpecsMap cacheSpecsMap, RedisConnectionFactory connectionFactory) {
-        super(CacheConstants.REPOSITORY_SETTINGS_CACHE, cacheSpecsMap, connectionFactory, configuration, new TbJavaRedisSerializer<>());
+        super(CacheConstants.REPOSITORY_SETTINGS_CACHE, cacheSpecsMap, connectionFactory, configuration, new TbJsonRedisSerializer<>(RepositorySettings.class));
```

Comment:
I strongly suggest for TbJsonRedisSerializer to check that the corresponding parameterized class has the ```@JsonIgnoreProperties(ignoreUnknown = true)``` annotation. This will save us from cases when some properties are renamed or deleted. Otherwise we will fail to decode the value even if it is valid. Please add this check and also add the corresponding annotations to all the classes that are used.